mmd_tools/operators/rigid_body.py

```diff
@@ -466,6 +466,12 @@ def __get_rigid_body_world_objects():
         if not rbw.constraints:
             rbw.constraints = bpy.data.collections.new('RigidBodyConstraints')
             rbw.constraints.use_fake_user = True
+
+        if bpy.app.version >= (2, 92, 0):
+            # see: https://github.com/powroupi/blender_mmd_tools/issues/333
+            bpy.context.scene.rigidbody_world.substeps_per_frame = 1
+            bpy.context.scene.rigidbody_world.solver_iterations = 60
```

Comment:
It might be better to keep default setting of `solver_iterations`, and only change `substeps_per_frame`.
```python
        if hasattr(bpy.context.scene.rigidbody_world, 'substeps_per_frame'):
            bpy.context.scene.rigidbody_world.substeps_per_frame = 1
```

Reply:
Thanks for the review.
If we edit the following function, we can apply it automatically when users import a MMD model.
Should we require users to perform additional operations?

https://github.com/UuuNyaa/blender_mmd_tools/blob/dfb3e225bbb0ad73799bfbdaa8629e69178ceda8/mmd_tools/core/rigid_body.py#L22

```python
def setRigidBodyWorldEnabled(enable):
    if bpy.ops.rigidbody.world_add.poll():
        bpy.ops.rigidbody.world_add()
    rigidbody_world = bpy.context.scene.rigidbody_world
    enabled = rigidbody_world.enabled
    rigidbody_world.enabled = enable
    return enabled
```

Reply:
If we apply it automatically, it would be troublesome when users want to tweak those values as they need. I think current change should be enough. :smile:

Reply:
I understand. Thank you.